Python/optimizer.c

```diff
@@ -602,13 +606,17 @@ translate_bytecode_to_trace(
                 goto done;
             }
         }
+        if (opcode == ENTER_EXECUTOR) {
```

Comment:
Why are we are done?
If we are wanting specialized traces, they may overlap.

Reply:
I've added a comment justifying the decision.